### PR TITLE
massive fix

### PR DESCRIPTION
**Pull request by rahadiankp**
@alfian853

### CODE REVIEW

File: project/proxy/proxy_server_api.py [hunk 1]
```diff
@@ -67,6 +78,35 @@ def ping(self):
...
+
+    def remove_server(self, server_uri):
+        print("Failure Detector has detected", server_uri, "is down")
+
+        # join pfd thread to main thread
+        # pfd_t = self.game_server_failure_detection_thread[server_uri]
+        # pfd_t.join()
+
+        self.server_api_list.remove(server_uri)
+        print("Unregistered -", server_uri)
```
Comment (alfian853) [inline project/proxy/proxy_server_api.py:96]: reset the last index to 0

File: project/proxy/proxy_server_api.py [hunk 2]
```diff
@@ -39,12 +48,14 @@ def push_command(self, command: str):
             return self.fail_response_no_server
 
         try:
-            server_response = self.server_api_list[last_index_call].push_command(command_data)
+            server_proxy_last_index = Pyro4.Proxy(self.server_api_list[last_index_call])
```
Comment (alfian853) [inline project/proxy/proxy_server_api.py]: don't use postfix _index for non integer

File: project/proxy/proxy_server_api.py (hunk 1 above)
Comment (rahadiankp) [reply project/proxy/proxy_server_api.py:96]: ok will do

File: project/proxy/proxy_server_api.py (hunk 2 above)
Comment (rahadiankp) [reply project/proxy/proxy_server_api.py]: ok will do

**Pull request merged**